src/main/java/org/baeldung/Application.java

```diff
@@ -1,4 +1,4 @@
-package org.baeldung.spring;
```

Comment:
https://github.com/Baeldung/spring-security-registration/pull/24/commits/92bdc5a6a4ab90d531f4b9d76e79517272dd2a8a this is saperate PR , I wished to open but unfortunately it got merged with  earlier one